src/mahoji/lib/abstracted_commands/alchCommand.ts

```diff
@@ -27,6 +27,7 @@ const unlimitedFireRuneProviders = resolveItems([
 
 // 5 tick action
 export const timePerAlch = Time.Second * 3;
+export const timePerAlchAgility = Time.Second * (3 + 10);
```

Comment:
Can you make it so the code that calculates the time also uses this same variable so theres not 2 places with this same value?